submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/service/SubmissionMongoPersistenceQueryService.kt

```diff
@@ -39,7 +39,7 @@ internal class SubmissionMongoPersistenceQueryService(
     }
 
     override fun findLatestExtByAccNo(accNo: String, includeFileListFiles: Boolean): ExtSubmission? {
-        val findByAccNo = submissionRepo.findLatestByAccNo(accNo)
+        val findByAccNo = submissionRepo.findByAccNo(accNo)
```

Comment:
why we have rename this?

Reply:
This wasn't renamed but replaced by a method that does exactly the same

https://github.com/EBIBioStudies/biostudies-backend-services/pull/596/files#diff-0e84528306be7e333417ccb2a28ec6909b7ff9bb50b4787ce5132980b518a3beR19